### ISSUE TITLE
[ui] one host, multiple urls

### ISSUE THREAD

**Issue by revolunet** [SocialGouv/dashlord#10]
report and config should handle multiple urls per host

**Comment by tristanrobert** [comment]
Why not a specific action that fetches urls from a product API? For instance, from https://github.com/betagouv/beta.gouv.fr/tree/master/content/_startups

**Comment by revolunet** [comment]
yes, anything that produce a list of urls and related repo could work, we just need to replace the "init" action : https://beta.gouv.fr/api/v2.1/startups.json

**Comment by tristanrobert** [comment]
> yes, anything that produce a list of urls and related repo could work, we just need to replace the "init" action : https://beta.gouv.fr/api/v2.1/startups.json

this API provides only one url (link) and one repository. It should be first upgraded to provide many urls and repositories.
